nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+
+const app = express();
+
+
+const addition=(req,res)=>{
+  const first=Number(req.query.first);
+  const second=Number(req.query.second)
+  const result=first+second;
+  if(('first' in req.query)&&('second' in req.query)){
```

Comment:
You've already grabbed these values from the query object, it's too late to check this here!